### PR TITLE
2024 Craig User Test Edits (3.21)

### PR DESCRIPTION
**Pull request by olehermanse**
Merge together: https://github.com/cfengine/masterfiles/pull/2836

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom try this in Jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/build-and-deploy-docs-3.21/5/)

**Jenkins:** https://ci.cfengine.com/job/build-and-deploy-docs-3.21/5/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-build-and-deploy-docs-3.21-5/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-build-and-deploy-docs-3.21-5/output/_site/

**Comment by craigcomstock** [comment]
jenkins jobs and documentation repos were mismatched in terms of which packages are built (by jenkins label) and which are searched for by documentation/generator/build/main.sh. jenkins jobs are adjusted to match documentation branches now and here's a rebuild:

[Build Status](https://ci.cfengine.com/job/build-and-deploy-docs-3.21/6/)

**Comment by craigcomstock** [comment]
@cf-bottom jenkins fast docs build please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/build-and-deploy-docs-3.21/7/)

**Jenkins:** https://ci.cfengine.com/job/build-and-deploy-docs-3.21/7/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-build-and-deploy-docs-3.21-7/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-build-and-deploy-docs-3.21-7/output/_site/

**Review by craigcomstock — APPROVED** [review]

File: getting-started/installation/secure-bootstrap.markdown [hunk 1]
```diff
@@ -41,7 +41,7 @@ settings. So after we have set-up our hub (using the standard procedure
 of `cf-agent --bootstrap $HUB_IP`) we take care of the following:
 
 * `cf-serverd` must never accept a connection from a client presenting an
-  untrusted key. [Disable automatic key trust][Masterfiles Policy Framework#trustkeysfrom]
+  untrusted key. [Disable automatic key trust][Masterfiles Policy Framework#Automatic bootstrap - Trusting keys from new hosts with trustkeysfrom]
```
Comment (craigcomstock) [inline getting-started/installation/secure-bootstrap.markdown:44]: the link here didn't work out, the generator doesn't complain but the link doesn't work. Looks like the generated link has one fewer spaces turned into dashes. I will fix with another commit to master and we can cherry pick back into this PR.

as generated:
```
http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-build-and-deploy-docs-3.21-7/output/_site/reference-masterfiles-policy-framework.html#automatic-bootstrap--trusting-keys-from-new-hosts-with-trustkeysfrom
```

what actually works:
```
http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-build-and-deploy-docs-3.21-7/output/_site/reference-masterfiles-policy-framework.html#automatic-bootstrap---trusting-keys-from-new-hosts-with-trustkeysfrom
```

**Review by craigcomstock — CHANGES_REQUESTED** [review]

**Pull request merged**